### DISCUSSION TITLE
🧩 FEATURE REQUEST: In budget option, add some note taking

### DISCUSSION THREAD

**Discussion by its-kundan** [dailydotdev/daily#1002] [Feature Requests]

**Comment by idoshamun** [comment]
Hi @its-kundan,

Can you elaborate please, not sure I understand?

**Comment by its-kundan** [comment]
Thanks for respond @idoshamun 
Most articles are  related to education. 
so, It would be better to take notes in own words.. for revision perpose.
( it gets difficult to re-read the same article ) .

**Comment by idoshamun** [comment]
Hi @its-kundan,

thanks for the suggestion :)
